.github/ISSUE_TEMPLATE/beta_feature.md

```diff
@@ -0,0 +1,58 @@
+---
+name: Beta Feature Requirements
+about: Requirements for promoting a feature to beta
+---
+
+This page lists the requirements for promoting a feature to beta. Please check off and document the steps as they are completed.
+
+**Feature Name:** 
+--- 
```

Comment:
could we highlight somehow which are new to beta (from alpha)?  Seems many are redundant.

Reply:
can we add link to demonstrate the feature has met the alpha approval?